.readthedocs.yaml

```diff
@@ -19,8 +19,9 @@ formats:
 
 # Optionally set the version of Python and requirements required to build your docs
 python:
-  version: 3.7
+  version: 3.8
   install:
+    - requirements: docs/readthedocs-requirements.txt
```

Comment:
This line is the fix: install `setuptools>=45.1.0` before installing the docs requirements (sphinx etc) and our package itself.